src/index.js

```diff
@@ -1,7 +1,9 @@
 import PropTypes from 'prop-types';
 import { render as preactRender, cloneElement as preactCloneElement, h, Component as PreactComponent, options } from 'preact';
 
-const version = '15.1.0'; // trick libraries to think we are react
+const version = '16.4.1'; // trick libraries to think we are react
```

Comment:
The react version number can only get changed once we adequately support the features from a given version.